### PR TITLE
Add missing Utils implementation for javascript

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: java/arcs/sdk/js/Utils.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// ktlint-disable filename
```
Comment (csilvestrini) [inline java/arcs/sdk/js/Utils.kt]: I think you can remove this. This file is actually named correctly!

File: java/arcs/sdk/js/Utils.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+@file:Suppress("PackageName", "TopLevelName")
```
Comment (csilvestrini) [inline java/arcs/sdk/js/Utils.kt]: I think you can remove TopLevelName here too

**Comment by shans** [comment]
(Travis failed due to timeout, ignoring the failure)

**Comment by shans** [comment]
Cloud build failure is a known flake

**Pull request merged**